### ISSUE TITLE
Random unicode character in C++ code?

### ISSUE THREAD

**Issue by that-ambuj** [feelfreelinux/octo4a#516]
https://github.com/feelfreelinux/octo4a/blob/3e9716a1f63c4c6e11ccbb5802070d9bb859f52a/app/app/src/main/cpp/vsp-pty.cpp#L14

I'm not sure if this will cause any issues or not but I don't think the character at the end is supposed be there.

**Comment by N07070** [comment]
Indeed, seems like a typo. You could send a pull request to remove it ?
